src/components/features/reviews-section/components/review-form.tsx

```
@@ -0,0 +1,187 @@
+
+    // Check authentication on mount
+    useEffect(() => {
+        // Function to check authentication status
+        const checkAuth = async () => {
+            try {
+                // Fetch token
+                const res = await fetch("/api/auth/token", { credentials: "include" });
+
+                // Handle non-OK responses
+                if (!res.ok) {
+                    console.warn("Failed to fetch token:", res.status);
```

ESLint

[error] Unexpected console statement.

(no-console)